### PR TITLE
Update docs to use secret refs instead of plaintext creds in TBS

### PR DESCRIPTION
**Pull request by tomkennedy513**
# Which other branches do you want a technical writer to cherry-pick this PR to (if any)?

All 1.5.x docs

### CODE REVIEW

File: tanzu-build-service/install-tbs.hbs.md [hunk 1]
```diff
@@ -335,40 +281,6 @@ To install `full` Tanzu Build Service dependencies:
 
     Where `VERSION` is the version of the Tanzu Build Service package you retrieved earlier.
 
-## <a id="auto-updates-config"></a> (Optional) Configure automatic dependency updates
```
Comment (ncarlson) [inline tanzu-build-service/install-tbs.hbs.md:338]: Removal of `<a id="auto-updates-config" />` will break the link on [line 29](https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/tanzu-build-service/install-tbs.hbs.md?plain=1#L29) of `install-tbs.hbs.md` (this file) and the link on [line 9](https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/multicluster/reference/tap-values-build-sample.hbs.md?plain=1#L94) of `multicluster/reference/tap-values-build-sample.hbs.md`.

File: tanzu-build-service/install-tbs.hbs.md [hunk 2]
```diff
@@ -154,74 +167,6 @@ to your `tbs-values.yaml` file. This is to exclude the default `lite` dependenci
...
-
-You can apply them directly in-line in plaintext in the `tbs-values.yaml` or `tap-values.yaml`
-configuration by using the `kp_default_repository_username`, `kp_default_repository_password`, `tanzunet_username`,
-and `tanzunet_password` fields.
-
-If you do not want credentials saved in plaintext, you can use existing secrets or IAM roles by using
-[secret references](#install-secret-refs) or [AWS IAM authentication](#tbs-tcli-install-ecr)
-in your `tbs-values.yaml` or `tap-values.yaml`.
-
-### <a id='install-secret-refs'></a> Use Secret references for registry credentials
```
Comment (ncarlson) [inline tanzu-build-service/install-tbs.hbs.md:169]: Link will break on [line 70](https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/multicluster/reference/tap-values-build-sample.hbs.md?plain=1#L70) of `multicluster/reference/tap-values-build-sample.hbs.md`.

Comment (ncarlson) [reply tanzu-build-service/install-tbs.hbs.md:169]: Here to: https://github.com/pivotal/docs-tap/blob/1ac74b9e0c554542a6a7eafd51026c67f4810f96/multicluster/reference/tap-values-iterate-sample.hbs.md?plain=1#L70

File: install-air-gap.hbs.md [hunk 3]
```diff
@@ -78,6 +78,32 @@ To relocate images from the VMware Tanzu Network registry to your air-gapped reg
...
+    ```console
+    tanzu secret registry add image-registry-creds \
+      --server "${REGISTRY_HOSTNAME}" \
+      --username "${REGISTRY_USERNAME}" \
+      --password "${REGISTRY_PASSWORD}" \
+      --namespace tap-install
+    ```
+
+   Where:
+    - `REGISTRY_HOST` is the hostname for the registry that will contain your writeable repository.
```
Comment (ncarlson) [inline install-air-gap.hbs.md:97]:
```suggestion
    - `REGISTRY_HOSTNAME` is the hostname for the registry that will contain your writeable repository.
```

File: install-air-gap.hbs.md [hunk 4]
```diff
@@ -78,6 +78,32 @@ To relocate images from the VMware Tanzu Network registry to your air-gapped reg
...
+      Examples:
+        - Harbor has the form `server: "my-harbor.io"`.
+        - Docker Hub has the form `server: "index.docker.io"`.
+        - Google Cloud Registry has the form `server: "gcr.io"`.
```
Comment (ncarlson) [inline install-air-gap.hbs.md:101]:
```suggestion
        - Harbor has the form `--server "my-harbor.io"`.
        - Docker Hub has the form `--server "index.docker.io"`.
        - Google Cloud Registry has the form `--server "gcr.io"`.
```

File: install-openshift.hbs.md [hunk 5]
```diff
@@ -78,6 +78,32 @@ To relocate images from the VMware Tanzu Network registry to your registry:
...
+    ```console
+    tanzu secret registry add image-registry-creds \
+      --server "${REGISTRY_HOSTNAME}" \
+      --username "${REGISTRY_USERNAME}" \
+      --password "${REGISTRY_PASSWORD}" \
+      --namespace tap-install
+    ```
+
+   Where:
+    - `REGISTRY_HOST` is the hostname for the registry that will contain your writeable repository.
```
Comment (ncarlson) [inline install-openshift.hbs.md:97]:
```suggestion
    - `REGISTRY_HOSTNAME` is the hostname for the registry that will contain your writeable repository.
```

File: tanzu-build-service/install-tbs.hbs.md (hunk 1 above)
Comment (tomkennedy513) [reply tanzu-build-service/install-tbs.hbs.md:338]: oh good catch i need to remove it from that file too

File: install-openshift.hbs.md [hunk 6]
```diff
@@ -78,6 +78,32 @@ To relocate images from the VMware Tanzu Network registry to your registry:
...
+      Examples:
+        - Harbor has the form `server: "my-harbor.io"`.
+        - Docker Hub has the form `server: "index.docker.io"`.
+        - Google Cloud Registry has the form `server: "gcr.io"`.
```
Comment (ncarlson) [inline install-openshift.hbs.md:101]:
```suggestion
        - Harbor has the form `--server "my-harbor.io"`.
        - Docker Hub has the form `--server "index.docker.io"`.
        - Google Cloud Registry has the form `--server "gcr.io"`.
```

File: install.hbs.md [hunk 7]
```diff
@@ -103,6 +103,32 @@ To add the Tanzu Application Platform package repository to your cluster:
...
+    ```console
+    tanzu secret registry add image-registry-creds \
+      --server "${REGISTRY_HOSTNAME}" \
+      --username "${REGISTRY_USERNAME}" \
+      --password "${REGISTRY_PASSWORD}" \
+      --namespace tap-install
+    ```
+
+    Where:
+    - `REGISTRY_HOST` is the hostname for the registry that will contain your writeable repository.
```
Comment (ncarlson) [inline install.hbs.md:122]:
```suggestion
    - `REGISTRY_HOSTNAME` is the hostname for the registry that will contain your writeable repository. 
```

File: install-air-gap.hbs.md [hunk 8]
```diff
@@ -234,8 +261,9 @@ profile: full
 ceip_policy_disclosed: true
 buildservice:
   kp_default_repository: "REPOSITORY"
-  kp_default_repository_username: "REGISTRY-USERNAME" # Takes the value from the shared section above by default, but can be overridden by setting a different value.
-  kp_default_repository_password: "REGISTRY-PASSWORD" # Takes the value from the shared section above by default, but can be overridden by setting a different value.
+  kp_default_repository_secret: # Takes the value from the shared section above by default, but can be overridden by setting a different value.
```
Comment (ncarlson) [inline install-air-gap.hbs.md:264]: Is it applicable (or worth including) this comment about the shared section on [line 413](https://github.com/pivotal/docs-tap/pull/2345/files#diff-ce6aea12f7588bb7a07de306c853a2071b7697c664e27a74f972ce488dac9cb4R413)

Comment (tomkennedy513) [reply install-air-gap.hbs.md:264]: ya we can

**Comment by YugangW** [comment]
Hi @tomkennedy513, Thanks for the PR. Would you mind helping us resolve the conflict so we can get it merged? Thanks.

**Comment by tomkennedy513** [comment]
> Hi @tomkennedy513, Thanks for the PR. Would you mind helping us resolve the conflict so we can get it merged? Thanks.

done!

**Pull request merged**